website/docs/d/nsxv_application.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxv_application"
+sidebar_current: "docs-vcd-data-source-nsxv-application"
+description: |-
+  Provides a VMware Cloud Director data source for reading NSX-V distributed firewall applications
+---
+
+# vcd\_nsxv\_application
+
+Provides a VMware Cloud Director data source for reading NSX-V distributed firewall applications
```

Comment:
Nit, it looks strange without a dot in this place.

```suggestion
Provides a VMware Cloud Director data source for reading NSX-V distributed firewall applications.
```

Reply:
fixed